network/messages_test.cc

```diff
@@ -19,138 +19,140 @@ limitations under the License.
+    ASSERT_EQ(request.GetMessageType(), MessageType::PM4_CAPTURE_REQUEST);
 
-    Network::Pm4CaptureResponse res_serialize;
+    Pm4CaptureResponse res_serialize;
     res_serialize.SetString("/sdcard/captures/dive_capture_0001.rd");
     buf.clear();
     status = res_serialize.Serialize(buf);
     ASSERT_TRUE(status.ok());
-    ASSERT_EQ(res_serialize.GetMessageType(), Network::MessageType::PM4_CAPTURE_RESPONSE);
-    Network::Pm4CaptureResponse res_deserialize;
+    ASSERT_EQ(res_serialize.GetMessageType(), MessageType::PM4_CAPTURE_RESPONSE);
+    Pm4CaptureResponse res_deserialize;
     status = res_deserialize.Deserialize(buf);
     ASSERT_TRUE(status.ok());
-    ASSERT_EQ(res_deserialize.GetMessageType(), Network::MessageType::PM4_CAPTURE_RESPONSE);
+    ASSERT_EQ(res_deserialize.GetMessageType(), MessageType::PM4_CAPTURE_RESPONSE);
     ASSERT_EQ(res_serialize.GetString(), res_deserialize.GetString());
 }
 
 TEST(MessagesTest, DownloadFileMessage)
 {
-    Network::DownloadFileRequest req_serialize;
+    DownloadFileRequest req_serialize;
     req_serialize.SetString("/sdcard/captures/dive_capture_0456.rd");
-    Network::Buffer buf;
-    auto            status = req_serialize.Serialize(buf);
+    Buffer buf;
+    auto   status = req_serialize.Serialize(buf);
     ASSERT_TRUE(status.ok());
-    ASSERT_EQ(req_serialize.GetMessageType(), Network::MessageType::DOWNLOAD_FILE_REQUEST);
-    Network::DownloadFileRequest req_deserialize;
+    ASSERT_EQ(req_serialize.GetMessageType(), MessageType::DOWNLOAD_FILE_REQUEST);
+    DownloadFileRequest req_deserialize;
     status = req_deserialize.Deserialize(buf);
     ASSERT_TRUE(status.ok());
-    ASSERT_EQ(req_deserialize.GetMessageType(), Network::MessageType::DOWNLOAD_FILE_REQUEST);
+    ASSERT_EQ(req_deserialize.GetMessageType(), MessageType::DOWNLOAD_FILE_REQUEST);
     ASSERT_EQ(req_serialize.GetString(), req_deserialize.GetString());
 
-    Network::DownloadFileResponse res_serialize;
+    DownloadFileResponse res_serialize;
     res_serialize.SetFound(false);
     res_serialize.SetErrorReason("File not found!");
     res_serialize.SetFilePath("/sdcard/captures/other_capture_0456.rd");
     res_serialize.SetFileSizeStr("234512340012367880000000000000");
     buf.clear();
     status = res_serialize.Serialize(buf);
     ASSERT_TRUE(status.ok());
-    ASSERT_EQ(res_serialize.GetMessageType(), Network::MessageType::DOWNLOAD_FILE_RESPONSE);
-    Network::DownloadFileResponse res_deserialize;
+    ASSERT_EQ(res_serialize.GetMessageType(), MessageType::DOWNLOAD_FILE_RESPONSE);
+    DownloadFileResponse res_deserialize;
     status = res_deserialize.Deserialize(buf);
     ASSERT_TRUE(status.ok());
-    ASSERT_EQ(res_deserialize.GetMessageType(), Network::MessageType::DOWNLOAD_FILE_RESPONSE);
+    ASSERT_EQ(res_deserialize.GetMessageType(), MessageType::DOWNLOAD_FILE_RESPONSE);
     ASSERT_EQ(res_serialize.GetFound(), res_deserialize.GetFound());
```

Comment:
curious would it be better to use EXPECT_ for GetFound()/GetErrorReason()/GetFileSizeStr()?